src/workerd/api/worker-rpc.h

```diff
@@ -20,6 +20,11 @@
 
 namespace workerd::api {
 
+// TODO(soon): This size limit is loosely based on Cap'n Proto's 'traversalLimitInWords'.
+// JS RPC does not support streaming messages yet, and until it does we should prevent
+// large messages from being sent.
+constexpr size_t MAX_JS_RPC_MESSAGE_SIZE = 32 * 1024 * 1024;
```

Comment:
We need to reduce this limit to like 1MB -- same as WebSocket, for the same reasons.